docs/source/bft_configuration.md

```diff
@@ -0,0 +1,228 @@
+# Configuring and operating a BFT ordering service
+
+**Audience**: * BFT ordering service admins*
+
+## Conceptual overview
+
+For a high level overview of the concept of ordering and how the supported
+ordering service implementations (including BFT) work at a high level, check
+out our conceptual documentation on the [Ordering Service](./orderer/ordering_service.md).
+
+To learn about the process of setting up an ordering node, check out our
+documentation on [Planning for an ordering service](./deployorderer/ordererplan.html).
+
+## Configuration
+
+A  BFT cluster is configured in two places:
+
+* **Local configuration**: Governs node specific aspects, such as TLS
+  communication, replication behavior, and file storage.
+
+* **Channel configuration**: Defines the membership of the  BFT cluster for the
+  corresponding channel, as well as protocol specific parameters such as timeouts.
+
+Unlike the [Raft ordering service](./raft_configuration.md), where nodes identify each other using TLS pinning, 
+BFT nodes identify each other using their enrollment certificate's private key.
+
+Each channel has its own instance of a BFT protocol running. Thus, each
+BFT node must be referenced in the configuration of each channel it participates in
+by adding its enrollment certificate (in `PEM` format) and MSP ID to the channel
+config. 
+
+The following [section](https://github.com/hyperledger/fabric/blob/main/sampleconfig/configtx.yaml) from `configtx.yaml` shows four BFT nodes (also called
+“consenters”) in the channel:
+
+```
+    ConsenterMapping:
+    - ID: 1
+      Host: bft0.example.com
+      Port: 7050
+      MSPID: OrdererOrg1
+      Identity: /path/to/identity
```

Comment:
Right now, this is an enrollment certificate. In the future there may be a different kind of identity that is not based on x509 certificates. The name selection is to be generic.

Reply:
Ok, so we'll leave the name as-is, but we still need to clarify that this points to the actual pem certificate file, rather than the msp directory as is common in other Fabric configs. A comment would suffice.

Reply:
done